app/components/sidebars/main/channels_list/channel_item/index.js

```diff
@@ -80,12 +84,13 @@ function makeMapStateToProps() {
             isChannelMuted: isChannelMuted(member),
             isGuest,
             isManualUnread: isManuallyUnread(state, ownProps.channelId),
-            mentions: member ? member.mention_count : 0,
+            mentions: member?.mention_count_root || 0,
```

Comment:
is this always mention_count_root even when collapse threads in not enabled?

Reply:
I don't think so. Good catch .. thank you 🙂